### PR TITLE
add testCase of TransactionToken. #463

### PR DESCRIPTION
**Pull request by btsaitoumsw**
Please review #463 .

@ikeyat 

---

https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/473/commits/8d9d46d3daf4bb150423584ad20172f8705ab564

add test cases with the additon of the test item. by @btsaitoumsw

### CODE REVIEW

File: terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/transactiontoken/TransactionTokenTest.java [hunk 1]
```diff
@@ -125,18 +125,35 @@ public void test02_01_normalBeginInEndScenario() {
...
+        newToken = driver.findElement(By.name("_TRANSACTION_TOKEN"))
+                .getAttribute("value");
+        assertThat(newToken, is(currentToken));
 
+        // keep
+        currentToken = newToken;
+        driver.findElement(By.id("btn-keep")).click();
+        assertThat(driver.findElement(By.name("_TRANSACTION_TOKEN")),
+                notNullValue());
+        newToken = driver.findElement(By.name("_TRANSACTION_TOKEN"))
```
Comment (btyoshikawaa) [inline terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/transactiontoken/TransactionTokenTest.java]: - [ ] Do you need to find same element 2 times ?

Comment (btyoshikawaa) [inline terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/transactiontoken/TransactionTokenTest.java]: @btsaitoumsw 
OK, Let's polish to another opportunity.

**Comment by ikeyat** [comment]
@btyoshikawaa 
Please merge after travis CI succeeds.

**Comment by iwanagah** [comment]
Please backport

**Pull request merged**